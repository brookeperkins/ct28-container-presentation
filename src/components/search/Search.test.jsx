import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Search from './Search';

describe('Search component', () => {
  afterEach(() => cleanup());
  const testSearch = function() {};

  it('renders the Search component', () => {
    const { asFragment } = render(<Search
      search="Test"
      onSubmit={testSearch}
      onChange={testSearch}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
