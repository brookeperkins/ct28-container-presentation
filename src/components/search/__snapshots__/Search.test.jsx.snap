// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders the Search component 1`] = `
<DocumentFragment>
  <label
    for="search"
  >
    Search
  </label>
  <input
    id="search"
    name="search"
    type="search"
    value="Test"
  />
  <button
    data-testid="button"
  >
    Search
  </button>
</DocumentFragment>
`;
